srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/Calculation.java

```diff
@@ -96,7 +102,40 @@ public void setSpecialty(final Specialty specialty)
                     "Cannot return list of variables because no specialty has been set.");
         }
         
-        return fSpecialty.getModelVariables();
+        final ImmutableSet<Variable> vars = fSpecialty.getModelVariables();
+        // Check our method contract.
+        if (hasMultipleProcedureVariables(vars))
+        {
+            // See method contract.
+            throw new ConfigurationException(
+                    "The specialty requires multiple ProcedureVariables");
```

Comment:
Shouldn't this read more like, "Specialties cannot have more than one ProcedureVariables" to make it clear what the error is trying to say.